src/openrct2/drawing/drawing.h

```diff
@@ -363,6 +363,8 @@ void shorten_path(utf8 *buffer, size_t bufferSize, const utf8 *path, sint32 avai
 void scrolling_text_initialise_bitmaps();
 sint32 scrolling_text_setup(rct_string_id stringId, uint16 scroll, uint16 scrollingMode);
 
+void FASTCALL gfx_get_sprite_size(sint32 image_id, sint32* width, sint32* height);
```

Comment:
The idea was to provide an automated way of retrieving an image size. Actually, come to think of it I remember why I added it. I believe I originally had the code directly within the window's paint function, but it said it needed to be a `FASTCALL` function.

I could try again and see if it works, or even just hardcode the image size into the layout for now, if you want. It's no biggie if we take it out.